### ISSUE TITLE
meta-review authorship

### ISSUE THREAD

**Issue by agitter** [greenelab/meta-review#65]
We recently reviewed all manubot and manubot-rootstock issues and pull requests to acknowledge all contributors.  @slochower and @vsmalladi your discussion and implementation (summarized below) substantially contributed to the Manubot design.  We would like to invite you to join as authors of this meta-review manuscript.  This manuscript discusses collaborative writing and Deep Review experience as an example.  It will also serve as the primary Manubot reference.

We are following [ICMJE authorship criteria](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html#two) so in addition to your previous contributions you would need to
- help draft or revise the manuscript
- approve the final manuscript
- agree to be accountable for the work

Please let us know if you would like to join as authors.  There are several open issues where you could help.

---
Contribution summary as of 2018-06-19
```
slochower:
Versioning discussion https://github.com/greenelab/manubot-rootstock/pull/101
Formatting change https://github.com/greenelab/manubot-rootstock/pull/56
Fix typos https://github.com/greenelab/manubot-rootstock/pull/25
Fix typo https://github.com/greenelab/manubot-rootstock/pull/22
Equation, figure, and table numbering https://github.com/greenelab/manubot-rootstock/pull/8
Setup discussion https://github.com/greenelab/manubot-rootstock/issues/6
Journal compatibility discussion https://github.com/greenelab/manubot-rootstock/issues/4
Web viewing discussion https://github.com/greenelab/manubot-rootstock/issues/58
Math rendering discussion https://github.com/greenelab/manubot-rootstock/issues/55
Build error report https://github.com/greenelab/manubot-rootstock/issues/47
PDF formatting discussion https://github.com/greenelab/manubot-rootstock/issues/20
SVG discussion https://github.com/greenelab/manubot-rootstock/issues/14
Citation dicussion https://github.com/greenelab/manubot-rootstock/issues/2
Automated numbering discussion https://github.com/greenelab/manubot-rootstock/issues/1
Citation errors https://github.com/greenelab/manubot/issues/33
Time zone discussion https://github.com/greenelab/manubot/issues/3

vsmalladi:
Setup instructions https://github.com/greenelab/manubot-rootstock/pull/84
Image width https://github.com/greenelab/manubot-rootstock/pull/79
CSL examples https://github.com/greenelab/manubot-rootstock/pull/46
Images in DOCX documentation https://github.com/greenelab/manubot-rootstock/pull/43/files
DOCX build https://github.com/greenelab/manubot-rootstock/pull/26
Check for missing author fields https://github.com/greenelab/manubot-rootstock/pull/17/files
Mac testing https://github.com/greenelab/manubot-rootstock/issues/74
PDF formatting discussion https://github.com/greenelab/manubot-rootstock/issues/20
SVG discussion https://github.com/greenelab/manubot-rootstock/issues/14
Feedback on initial Manubot code https://github.com/greenelab/manubot/pull/2
```

**Comment by slochower** [comment]
In principle, I'm in. But I'd like to read through the current manuscript first, and I have a couple of questions.

1. What's the timeline for finishing the writing and submitting? If this is a priority *right now* then I probably won't be able to make time for it. But otherwise, I think I can make it work.
2. What's the goal of this manuscript? Is it to raise awareness of `manubot`, discuss collaborative writing, take a philosophical position on open science, all of the above, something different?

**Comment by agitter** [comment]
1. Last week we resumed actively working on the manuscript with the intention of submitting ASAP (i.e. 1-2 weeks).  There isn't a hard deadline though, so our timeline is open for discussion.
2. All of the above.  After writing Deep Review, we received interest in the collaborative process and what lessons we learned from it.  Our goals here are to present the benefits of open collaborative writing and how Manubot offers important and unique features to support collaborative writing.

**Comment by slochower** [comment]
Looks good to me. How do you want to proceed?

**Comment by vsmalladi** [comment]
I am also in. How do we proceed.

**Comment by agitter** [comment]
I suggest that you first read through the current draft and open issues or pull requests for anything you would like discuss or edit.

Some of the open issues we could use help with are #44 (especially verifying the information in the linked table), #47, and #66.

**Comment by agitter** [comment]
I'd like to submit this soon and am re-engaging in our open issues.

@vsmalladi and @slochower can you please add yourselves to the authors in `metadata.yaml` once you have approved the draft?  We could also still use help with #44, #47, and #66, which would help us justify including you as authors but not others who have made smaller Manubot contributions.

Based on the contribution summary above, I propose the author order:
- @dhimmel 
- @slochower 
- @vsmalladi 
- @cgreene 
- @agitter

Any objections to that order?

**Comment by slochower** [comment]
Thanks @agitter. I made a PR for the authoring to get the ball rolling on that. I'm on-and-off traveling for the next 36 hours; I can help address the issues you linked on Sunday.

**Comment by vsmalladi** [comment]
Thanks @agitter. I am also traveling over the next couple of days with limited access but look at the issues when I have a more stable connection later this weekend.

**Comment by agitter** [comment]
@vsmalladi and @slochower, I plan to block off time this Friday to work on issues and review open pull requests.  If you have any revisions in progress, could you please submit a pull request by Friday?

**Comment by slochower** [comment]
@agitter I didn't have time to address https://github.com/greenelab/meta-review/issues/47 (I don't have any work in progress aside from the comments made in that thread), do you want me to take a crack at it tomorrow sometime or should I wait until you give the manuscript a fresh coat of paint?

**Comment by vsmalladi** [comment]
@agitter I just got back from travels and am looking at the outstanding issues. I can try to take a crack at #66. Should I wait until the manuscript has merged in the current outstanding updates before trying to integrate these references?

**Comment by agitter** [comment]
We don't have to merge all open pull requests before updating other parts of the manuscript, but I don't think we should have two open pull requests editing the same text.  If you have suggestions for #66, can you please add them in a review of #81?

**Comment by vsmalladi** [comment]
@agitter will do.

**Comment by agitter** [comment]
Closed by #75 and #79
